sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -457,6 +457,11 @@ public CaseFacadeEjb(CaseService service, UserService userService) {
 		super(Case.class, CaseDataDto.class, service, userService);
 	}
 
+	@Override
+	public AbstractCoreAdoService<Case> getEntityService() {
```

Comment:
Same here (see Barna's comment)

Reply:
removed along with the similar method from other core services